lib/mrkt/concerns/crud_leads.rb

```diff
@@ -40,5 +40,17 @@ def associate_lead(id, cookie)
         json_payload(req, {})
       end
     end
+
+    def merge_leads(winning_lead_id, losing_lead_ids, merge_in_crm: false)
+      params = Faraday::Utils::ParamsHash.new
```

Comment:
There is no need to use `Faraday::Utils::ParamsHash.new` a simple Hash would make it.